dev-python/gitdb2/gitdb2-2.0.2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="GitDB is a pure-Python git object database"
+HOMEPAGE="
+	https://github.com/gitpython-developers/gitdb
+	https://pypi.python.org/pypi/gitdb2"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	!dev-python/gitdb[${PYTHON_USEDEP}]
+	>=dev-python/smmap2-2.0.1[${PYTHON_USEDEP}]"
+DEPEND="${RDEPEND}"
+# Testsuite require files from a git repo
```

Comment:
Actually, you need a clone, as some packed data is referenced in tests:
https://github.com/gitpython-developers/gitdb/blob/master/gitdb/test/test_pack.py#L49

Just cding into the source directory and running git init unfortunatly isn't enough. Tests still fail with e.g:

```
======================================================================
ERROR: test_pack_writing (gitdb.test.performance.test_pack_streaming.TestPackStreamingPerformance)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/gitdb-2.0.2/gitdb/test/lib.py", line 72, in wrapper
    return func(self, *args, **kwargs)
  File "/tmp/gitdb-2.0.2/gitdb/test/performance/test_pack_streaming.py", line 58, in test_pack_writing
    PackEntity.write_pack((pdb.stream(sha) for sha in pdb.sha_iter()), ostream.write, object_count=ni)
  File "/tmp/gitdb-2.0.2/gitdb/pack.py", line 992, in write_pack
    "Expected to write %i objects into pack, but received only %i from iterators" % (object_count, actual_count))
nose.proxy.ValueError: Expected to write 1000 objects into pack, but received only 0 from iterators
-------------------- >> begin captured logging << --------------------
root: INFO: You can set the GITDB_TEST_GIT_REPO_BASE environment variable to a .git repository of your choice - defaulting to the gitdb repository
--------------------- >> end captured logging << ---------------------
```

Running the tests from a clone, tests succeed. I assume this is also the reason why tests are disabled within `dev-python/gitdb`.